### PR TITLE
refactor/camelcase, semicolons

### PR DESCRIPTION
**Pull request by DoubleCouponDay**

### CODE REVIEW

File: getDataFile/jdoCases.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 const urlAdapter = require("./generic/url");
 const MOJconstants = require('../constants/MOJresponse');
 const constants = require('../constants');
-const casemodel = require('../models/case');
-const commonfuncs = require('../common/functions');
+const caseModel = require('../models/case');
+const commonFuncs = require('../common/functions');
```
Comment (williamparry) [inline getDataFile/jdoCases.js]: Why is this not called helpers like before?

Comment (DoubleCouponDay) [reply getDataFile/jdoCases.js]: I didn't think it mattered.

File: common/setup.js [hunk 2]
```diff
@@ -60,7 +61,7 @@ const setup = async (envFileName, resumeSessionId = 0) => {
         port: process.env[constEnv.port],
         user: process.env[constEnv.dbUser],
         password: process.env[constEnv.dbPass],
-        client_encoding: 'UTF8'
+        client_encoding: 'UTF8' //eslint warns but this case is required to interface with pg
```
[the remark this answers is not in the document]
Comment (DoubleCouponDay) [reply common/setup.js]: fixed.

File: getDataFile/jdoCases.js (hunk 1 above)
Comment (williamparry) [reply getDataFile/jdoCases.js]: Inconsistency is tech-debt.

File: common/setup.js [hunk 3]
```diff
@@ -72,25 +73,25 @@ const setup = async (envFileName, resumeSessionId = 0) => {
...
 module.exports.getStartData = setup;
 
-module.exports.startApplication = function(entrypoint) {
+module.exports.startApplication = function(entryPoint) {
     const argv = yargs.argv;
 
     let runner = async () => {
-        setupdata = await setup(argv.env); 
-        const {pgPoolConnection, pgPromise} = setupdata;
+        let setupData = await setup(argv.env);
```
[the remark this answers is not in the document]
Comment (DoubleCouponDay) [reply common/setup.js]: refactored.

File: getDataFile/jdoCases.js (hunk 1 above)
Comment (DoubleCouponDay) [reply getDataFile/jdoCases.js]: refactored inconsistent naming.

**Comment by williamparry** [comment]
Great work

**Pull request merged**